### PR TITLE
New commit as test

### PR DESCRIPTION
**Pull request by N-Yog**
Please review

### CODE REVIEW

File: JavaTutorial/JavaTutorial/src/main/java/com/tutorial/collections/CarObjectDemo2.java [hunk 1]
```diff
@@ -0,0 +1,25 @@
+/**
+ * 
+ */
+package com.tutorial.collections;
+
+/**
```
Comment (Y-Naik) [inline JavaTutorial/JavaTutorial/src/main/java/com/tutorial/collections/CarObjectDemo2.java:6]: be specific

File: JavaTutorial/JavaTutorial/src/main/java/com/tutorial/collections/CarObjectDemo2.java [hunk 2]
```diff
@@ -0,0 +1,25 @@
+/**
+ * 
+ */
+package com.tutorial.collections;
```
Comment (Y-Naik) [inline JavaTutorial/JavaTutorial/src/main/java/com/tutorial/collections/CarObjectDemo2.java:4]: add comments to file

**Review by Y-Naik — COMMENTED** [review]
complete

**Comment by N-Yog** [comment]
have added additional comments

**Review by Y-Naik — APPROVED** [review]
changes accepted

**Pull request merged**